Source/Dafny/Verifier/Translator.ClassMembers.cs

```diff
@@ -645,7 +645,7 @@ public partial class Translator {
         m.Outs.Iter(p => AddExistingDefiniteAssignmentTracker(p, m.IsGhost));
         // translate the body
         TrStmt(m.Body, builder, localVariables, etran);
-        m.Outs.Iter(p => CheckDefiniteAssignmentReturn(m.BodyEndTok, p, builder));
+        m.Outs.Iter(p => CheckDefiniteAssignmentReturn(m.Body.EndTok, p, builder));
```

Comment:
Wow... but why is this solving the problem? Won't you have a problem if body is null?

Reply:
The previous call (and some of the following code) would also crash if the body was null, so I believe it's not possible at this point in the code path. If it is, that's a separate thing to fix. :)